### DISCUSSION TITLE
How to lower the 0.512 MeV limit on AE to 1 keV

### DISCUSSION THREAD

**Discussion by barghashr** [nrc-cnrc/EGSnrc#935] [Questions] [answered]
I am trying to simulate a 50 keV x-ray treatment machine but I am not getting accurate results.  I get a low Percent Depth Dose (PDD) when compared to the real measured PDD.  I have all the MC transport parameters set to give the most accurate calculations as follows: 
(Please correct me if I have any of the parameters wrong)

```
 Global ECUT= 0.001
 Global PCUT= 0.001
 Global SMAX= 1e10
 ESTEPE= 0.1
 XIMAX= 0.5
 Boundary crossing algorithm= EXACT
 Skin depth for BCA= 0
 Electron-step algorithm= PRESTA-II
 Spin effects= On
 Brems angular sampling= KM
 Brems cross sections= NRC
 Bound Compton scattering= Norej
 Compton cross sections= default
 Pair angular sampling= KM
 Pair cross sections= NRC
 Photoelectron angular sampling= On
 Rayleigh scattering= On
 Atomic relaxations= EADL
 Electron impact ionization= On
 Photon cross sections= xcom
 Photon cross-sections output= Off
```

I am running the simulation in Pegsless mode to be able to set AE and AP to 1 keV, but I am not able to set AE to as low as 1 keV.  I believe the problem is that the lowest AE is limited to 0.512 MeV and I cannot go lower than that.  How can I lower the 0.512 MeV limit for AE down to 1 keV?

**Comment by ftessier** [comment, marked as answer]
In EGSnrc, the electron cutoff energy is specified as the **total energy, including the electron rest mass of about 0.511 MeV**, hence a cutoff energy of 0.512 MeV is effectively a 1 keV cutoff on the kinetic energy of the electron, and is the minimum allowed value.

**Reply by barghashr** [reply]
I see, thank you very much for the clarification.
